### DISCUSSION TITLE
[5.2] Usability: Media Manager > Save and new

### DISCUSSION THREAD

**Discussion by micker** [joomla/joomla-cms#43593] [General]
Hello editing image in media manager is realy great adding but i think we can have a better workflow with a save and new function
1 "save" is to save over the original image
2 "save a new image" can create a new image with all modification
what do you think @woluweb ?

**Comment by rdeutz** [comment]
Please discuss first. Issues are for bugs or for feature requests with full instructions how to implement.

**Reply by micker** [reply]
Yep i am agree.
